Change table alias for consistency

jtpalmer opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Description
<!--- Describe your changes in detail -->
<!--- Include screenshots for GUI changes if appropriate -->
<!--- If any documentation outside of this repo needs to be added or updated,
      please include links to the relevant docs and how they should be changed -->

Changes the alias used for the aggregate table in the jobs realm `JobDataset` query.  

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->

The group by refactoring changed the table alias from `jf` to `agg`.  This file was not changed so it was still using `jf` as an associative array key in the PHP code even though the generated SQL is now using `agg`.  The change makes it clear that `agg` in the PHP code is the same `agg` in the corresponding SQL.  It also prevents another table from being aliased to `agg` and allowing other table to used `jf` as an alias if necessary.

## Tests performed
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->

Tested manually.

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] My code follows the code style of this project as found in the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

jtpalmer commented:
This also fixes an issue created by ubccr/xdmod-xsede#209 (which also changes `jf` to `agg`).

plessbd commented:
Yeah the refactor changed the alias to be "more readable", for some of the supremm stuff it did not change (yet) to make the refactor code easier to review (and because the supremm stuff was auto generated).  Sorry if I missed some.

jtpalmer commented:
This specific change wasn't actually required by the other changes (because `jf` was only being used as a key in a PHP array.), but when I was changing the XSEDE realms classes (which really did need to change because they contained a SQL fragment that used `jf`) I changed all the realms.  Since I changed the XSEDE jobs realm I broke it (because I didn't change the PHP in this repository).  It's more confusing than anything to be using `jf` in the PHP classes, but `agg` in the generated SQL.  That's my primary motivation in making this change (and that since there was and XSEDE class that did need to change).  I just added my previous comment to create a link between the pull requests and for future reference.
